### ISSUE TITLE
Upon Navigating to the Solid Square, the Narrator did not Announce on the Solid Square.

### ISSUE THREAD

**Issue by msftedad** [microsoft/azuredatastudio#23944]
- Azure Data Studio Version: 
- OS Version: Windows 11 Enterprise

Steps to Reproduce:

1. Navigate to this link [First Level Child (zorana-jankovic.github.io)](https://zorana-jankovic.github.io/example/#/start)
2. Navigate to solid square. 
3. Observe the narrator's announcement.

Does this issue occur when all extensions are disabled?: Yes/No
Yes
![204941_Docsify-Web](https://github.com/microsoft/azuredatastudio/assets/93735775/d2b6b0a7-7932-4ac6-899b-410a22e0f727)
[Solid square screen reader attachment.webm](https://github.com/microsoft/azuredatastudio/assets/93735775/a9ddb21c-5f5d-4ee8-92b7-a5f40d1ce791)

**Comment by corivera** [comment]
Closing this issue since it looks like this a github website issue and is unrelated to ADS.
